view/concept-shared.twig

```diff
@@ -42,7 +42,16 @@
       {% spaceless %}
       <div class="row{% if concept.type == 'skosext:DeprecatedConcept' %} deprecated{% endif %} property prop-preflabel">
         <div class="property-label property-label-pref">
-          <h3 class="versal">{% set subPrefLabelTranslation = concept.preferredSubpropertyLabelTranslation(request.lang) %}{% if subPrefLabelTranslation %}{{ subPrefLabelTranslation }}{% else %}{{ 'skos:prefLabel'|trans }}{% endif %}</h3>
+          <h3 class="versal">
+            {% set subPrefLabelTranslation = concept.preferredSubpropertyLabelTranslation(request.lang) %}
+            {% if subPrefLabelTranslation %}
+              {{ subPrefLabelTranslation }}
+            {% elseif custom_labels['skos:prefLabel'] %}
```

Comment:
This seems fragile - it's possible that *some* overrides are defined for `skos:prefLabel`, but not labels (only tooltips), or not in the right language. The test should be more specific and match the expression on the next line.

Reply:
Fixed in a new commit